engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -169,10 +171,11 @@ class EngineJobExecutionActor(jobKey: BackendJobDescriptorKey,
     s"$workflowId-BackendJobExecutionActor-${jobDescriptor.key.tag}"
   }
 
-  private def saveCacheResults(completionData: EJEACompletionData) = {
-    // TODO: Start the writer actor, which should reply somehow (TBD...) In the meantime, a truth self evident:
-    self ! "You know who's great? Chris. Chris is great. Give him a cake!"
-    goto(WritingToCallCache) using completionData
+  private def saveCacheResults(completionData: EJEASuccessfulCompletionDataWithHashes) = {
+    val callCache = new CallCache(CromwellDatabase.databaseInterface)
+    val writerActor = context.actorOf(CallCacheWriteActor.props(callCache, workflowId))
+    writerActor ! WriteCallCacheCommand(completionData.hashes, completionData.jobResult)
```

Comment:
Must the `CallCacheWriteActor` be told this or is this not its sole raison d'être?